core/src/test/java/org/elasticsearch/search/aggregations/bucket/significant/InternalSignificantTermsTestCase.java

```diff
@@ -91,14 +143,22 @@ protected void assertBucket(MultiBucketsAggregation.Bucket expected, MultiBucket
 
         assertEquals(expectedSigTerm.getSignificanceScore(), actualSigTerm.getSignificanceScore(), 0.0);
         assertEquals(expectedSigTerm.getSubsetDf(), actualSigTerm.getSubsetDf());
+        assertEquals(actualSigTerm.getDocCount(), actualSigTerm.getSubsetDf());
```

Comment:
Thanks for the clarification, that makes sense. Can you maybe add a comparison between either `expectedSigTerm.getDocCount(), actualSigTerm.getSubsetDf()` or  `expectedSigTerm.getDocCount(), expectedSigTerm.getSubsetDf()`, then the explanation you give here would be clearer from the test.